core/nodejs14Action/Dockerfile

```diff
@@ -17,60 +17,49 @@
 
 # build go proxy from source
 ARG GO_PROXY_BASE_IMAGE=golang:1.18
-FROM $GO_PROXY_BASE_IMAGE AS builder_source
+FROM $GO_PROXY_BASE_IMAGE AS builder
+
 ARG GO_PROXY_GITHUB_USER=nimbella-corp
 ARG GO_PROXY_GITHUB_BRANCH=dev
-RUN git clone --branch ${GO_PROXY_GITHUB_BRANCH} \
```

Comment:
why remove these - will will make it harder when we rebase on upstream/apache as we could end up with conflicts?

Reply:
This is a longer discussion I save for a higher bandwidth medium.  For now: Will back this out, it's only a few MB worth of saving. Most of the saving is in the slim image.

Reply:
Aaaaactually, these lines aren't even upstream. We changed Node.js to be an actionloop downstream, so they don't even include the go proxy at all. I think this is fine as-is then.